photon-server/src/main/java/org/photonvision/common/dataflow/networktables/NTDataChangeListener.java

```diff
@@ -0,0 +1,5 @@
+package org.photonvision.common.dataflow.networktables;
+
+public class NTDataProvider {
+
+}
```

Comment:
What's this class for?

Reply:
That must have slipped in. It's getting renamed `NTDataChangeListener` and will house the listeners for `pipelineIndex`, `driverMode`, etc